src/screens/MessageScreen.tsx

```diff
@@ -9,13 +9,8 @@ import { t } from 'i18next';
 import { ActivityIndicatorView } from '../components/ActivityIndicatorView';
 import { tID } from '../common/testID';
 import { useMyMessages } from '../hooks/useMyMessages';
-import {
-  format,
-  differenceInMinutes,
-  differenceInHours,
-  differenceInDays,
-  differenceInSeconds,
-} from 'date-fns';
+import { format, differenceInDays } from 'date-fns';
+import dayjs from 'dayjs';
```

Comment:
My two cents here are that it'd be great if we could consolidate on a single date library for the SDK. I think that means that keeping your previous code that manually calculated the "from now" format while only keeping `date-fns` looks better than having both `date-fns` and `dayjs` at the same time just for `dayjs`'s format helper.

Reply:
I wonder if we should just swap date-fns with dayjs. I think date-fns doesn't have nearly as much utility and we are not too deep with date-fns yet.

Reply:
I think for now I'll backpedal on the dayjs decision and use the function I initially created.

Reply:
+1, sounds good. If it does come around, I'm personally in favor of `dayjs`. I find it's much more ergonomic than `date-fns`.

Reply:
Done, switched back to a function.